.gitignore

```diff
@@ -12,6 +12,7 @@ _deps
 sources/
 .vscode/
 *.venv
+*venv
```

Comment:
This is a reasonable change (had the same in mind earlier!) but should not be part of this PR. Please send a separate patch.